pet-store-vanilla-js/utils.js

```diff
@@ -0,0 +1,22 @@
+export function isNullOrWhitespace(value) {
+    if (value == null || value.trim().length == 0) {
```

Comment:
* Use `===` instead of `==` when possible.
* You can simplify this function to `return value == null || value.trim().length == 0;`.
* You don't need this function at all. Use `!value` instead.

Reply:
If i remove the function and only use `!value`, white spaces are valid.
I just simplified it to `return value == null || value.trim().length == 0;`